### PR TITLE
quickfix on ns->shema

### PR DESCRIPTION
**Pull request by mvandervliet**

### CODE REVIEW

File: deploy/complete/helm-chart/mushop/charts/carts/templates/oadb-secrets.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{- $usesOsb := (index (.Values.global | default .Values) "osb").atp | default .Values.osb.atp -}}
 {{- $oadbService := .Values.secrets.oadbService | default .Values.global.oadbService -}}
 {{- $oadbWalletPw := .Values.secrets.oadbWalletPassword | default .Values.global.oadbWalletPassword -}}
-{{- $oadbUser := .Values.secrets.oadbUser | default (printf "%s_%s_user" .Release.Namespace .Chart.Name ) | trunc 96 -}}
+{{- $oadbUser := .Values.secrets.oadbUser | default (printf "%s_%s_user" .Release.Namespace .Chart.Name ) | replace "-" "_" | trunc 96 -}}
```
Comment (junior) [inline deploy/complete/helm-chart/mushop/charts/carts/templates/oadb-secrets.yaml:4]: replacing affects any other place? the user will not be confused that tried something like mushop-something and end up with mushop_something?

**Comment by jeevanjoseph** [comment]
Oracle db objects also have a 30 byte limitation in names- so maybe also
trunc the names ?

https://docs.oracle.com/cd/B19306_01/server.102/b14200/sql_elements008.htm


-- 
Sent from Gmail Mobile

File: deploy/complete/helm-chart/mushop/charts/carts/templates/oadb-secrets.yaml (hunk 1 above)
Comment (mvandervliet) [reply deploy/complete/helm-chart/mushop/charts/carts/templates/oadb-secrets.yaml:4]: Nah, usually we don’t even go to that level of depth

**Comment by mvandervliet** [comment]
> Oracle db objects also have a 30 byte limitation in names- so maybe also trunc the names to that length ? https://docs.oracle.com/cd/B19306_01/server.102/b14200/sql_elements008.htm
> […](#)

It’s already truncated but to 96.. I thought this was increased in a recent version

**Comment by junior** [comment]
> > Oracle db objects also have a 30 byte limitation in names- so maybe also trunc the names to that length ? https://docs.oracle.com/cd/B19306_01/server.102/b14200/sql_elements008.htm
> > […](#)
> 
> It’s already truncated but to 96.. I thought this was increased in a recent version

Yeah. 30 was the old 10g's. 12g, 18c, 19c+ are 96

**Pull request merged**